.codecov.yml

```diff
@@ -314,18 +314,18 @@ coverage:
         target: 75
         flags:
         - kong
-      KubeVirt_Handler:
+      KubeVirt_API:
         target: 75
         flags:
-        - kubevirt_handler
+        - kubevirt_api
       KubeVirt_Controller:
         target: 75
         flags:
         - kubevirt_controller
-      KubeVirt_API:
+      KubeVirt_Handler:
         target: 75
         flags:
-        - kubevirt_api
+        - kubevirt_handler
```

Comment:
When was the last time you updated this branch? I think slurm also had these changes when it was merged.

https://github.com/DataDog/integrations-core/pull/18893/files

Seems like it's already there in master:
https://github.com/DataDog/integrations-core/blob/master/.codecov.yml#L317-L328